testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -120,6 +121,51 @@ def test_write_coordinates(self, u, tmpdir):
         with tmpdir.as_cwd():
             u.atoms.write("test.xtc")
 
+    @pytest.mark.parametrize('frames', (
```

Comment:
`None` is missing from the test. Should be tested explicitly. Ideally for multiple writers.

Reply:
Added in a different test.

Reply:
ok